src/interactive/report-item.tsx

```diff
@@ -20,11 +20,22 @@ export const ReportItemComponent = () => {
       addGetReportItemAnswerListener((request) => {
         // TODO: update lara interactive api to change addGetReportItemAnswerListener to a generic with <IInteractiveState, IAuthoredState>
         // and remove the `any` after request
-        const {type, platformUserId, interactiveState, authoredState} = request as any;
+        const { type, platformUserId, interactiveState, authoredState, version } = request as any;
         switch (type) {
           case "html":
```

Comment:
This `type` on the request is a legacy property:
https://github.com/concord-consortium/portal-report/blob/4b57738e978f508f4843807a7af3549467224baa/js/reducers/report-reducer.ts#L136-L139
So it would be better to just ignore the type.

It would also be good if you checked the version of the request incase we change the format again. An example of that is here: https://github.com/concord-consortium/lara/blob/abc996a1ea45b229e2a8b7b3dca1777261f9daa5/lara-typescript/src/example-interactives/src/attachments/report-item.tsx#L20

Reply:
Thanks for catching the unnecessary `type` switch. I removed that and added the version check, too, in the latest commit.